dash_labs/plugins/pages.py

```diff
@@ -224,16 +241,17 @@ def _filename_to_name(filename):
     return filename.split(".")[-1].replace("_", " ").capitalize()
 
 
-def _filename_to_path(filename):
-    return filename.replace("_", "-").replace(".", "/").lower().split("pages")[-1]
-
+def _infer_path(filename, template):
+    if template is None:
+        path = filename.replace("_", "-").replace(".", "/").lower().split("pages")[-1]
+        path = "/" + path if not path.startswith("/") else path
+        return path
+    else:
+        # replace the variables in the template with "None"
+        return re.sub("<.+?>", "none", template)
```

Comment:
```suggestion
        return re.sub("<[^<>]+>", "none", template)
```
Not a big deal, I'm just brainstorming ways the user could break this, and at present unmatched or empty angle brackets would produce very weird results.

Maybe in fact the best way to handle this would be validating the template in `register_page`: break it into segments like we do in `_parse_path_variables`, but then for any segment that has either `<` or `>` in the segment, throw an error if the segment doesn't match `^<[^<>]+>$`

Reply:
ah, yes - good idea. I'll add that